packages/router-core/src/utils.ts

```diff
@@ -488,3 +488,57 @@ export function findLast<T>(
   }
   return undefined
 }
+
+const DECODE_IGNORE_LIST = [
```

Comment:
Although it'll be negligible, could we precompute this into a Set? Perhaps even have them in their encoded form already?